### PR TITLE
Fix changed version line

### PR DESCRIPTION
**Pull request by Toasterson**
https://github.com/illumos/illumos-gate/commit/4125432b52bd46f8b4129eae0625c71e159f0077

### CODE REVIEW

**Review by Bill-Sommerfeld — COMMENTED** [review]
While this would work and accomplish what you want, it would be more robust to pass -V $(...) to the invocation of `nightly` below on line 109.

**Comment by mtelka** [comment]
@Toasterson, while touching the `illumos-gate` component, please update the package metadata (`gmake update-metadata`) too.  Thank you.

**Comment by Toasterson** [comment]
@Bill-Sommerfeld is there somehow a filter for '-V' in illumos.sh? https://jenkins.openindiana.aurora-opencloud.org/blue/organizations/jenkins/OpenIndiana%20Organisation%2Foi-userland/detail/PR-18701/5/pipeline

File: components/openindiana/illumos-gate/Makefile [hunk 1]
```diff
@@ -106,7 +105,7 @@ $(BUILD_DIR)/$(MACH)/.built: $(SOURCE_DIR)/.patched $(BUILD_DIR)/runtime-perl.p5
 	  echo export PKGVERS_BRANCH=$(ONNV_BUILDNUM); \
 	  echo export BOOTBANNER1=\"$(DISTRIBUTION_NAME) $(DISTRIBUTION_VERSION) Version ^v ^w-bit\") > \
 	  illumos.sh && \
-	  time $(ENV) -i ./usr/src/tools/scripts/nightly illumos.sh
+	  time $(ENV) ./usr/src/tools/scripts/nightly illumos.sh -V $(shell git log -1 --format=illumos-%h)
```
Comment (mtelka) [inline components/openindiana/illumos-gate/Makefile]: Why removal of `-i` here?

**Comment by Bill-Sommerfeld** [comment]
> @Bill-Sommerfeld is there somehow a filter for '-V' in illumos.sh? https://jenkins.openindiana.aurora-opencloud.org/blue/organizations/jenkins/OpenIndiana%20Organisation%2Foi-userland/detail/PR-18701/5/pipeline

Command-line options to `nightly` need to come before the name of the environment file.

Try in this order:

```
.../nightly -V $(...) illumos.sh
```

File: components/openindiana/illumos-gate/Makefile (hunk 1 above)
Comment (Toasterson) [reply components/openindiana/illumos-gate/Makefile]: There is already an '-i' in there apparently. It's visible in the logs.

**Comment by Toasterson** [comment]
Thanks that worked.

File: components/openindiana/illumos-gate/Makefile (hunk 1 above)
Comment (mtelka) [reply components/openindiana/illumos-gate/Makefile]: Ah, you are right.  `ENV` got changed in January to contain `-i`. :-)

**Pull request merged**